README.md

```diff
@@ -16,6 +16,38 @@ are available as library to use in multiple usecases. These could be:
 If you want to build the command locally, you just need Golang version 1.20 and run `go build` on the root level of
 this repository.
 
+## Git pre-commit hook
+
+It is possible to add quality checks as git pre-commit hook in the repository which will run checks triggered by git commit command. To achieve that please follow below steps:
+
+1. Download latest version of the tractusx-quality-checks tool for your platform under Asset section at https://github.com/eclipse-tractusx/tractusx-quality-checks/releases
```

Comment:
Maybe add direct url to the latest release.

```suggestion
1. Download latest version of the tractusx-quality-checks tool for your platform under Asset section at https://github.com/eclipse-tractusx/tractusx-quality-checks/latest
```

Reply:
good point, thanks!